### DISCUSSION TITLE
Is it possible to manually track Time Sheets?

### DISCUSSION THREAD

**Discussion by abhuva** [Leantime/leantime#607] [Q&A] [answered]
We are currently trying out Leantime for our project-management. Lots of our work is not done at the computer (lots of physical activities), and i wonder if its possible to manually input times, instead of using the timer from the kanban board view.

**Comment by marcelfolaron** [comment, marked as answer]
Hi,

Yes, you should see a "My Timesheets" menu point in the top menu. That
allows you to change and fill in timesheets for a whole week.
